docs/docs/developers/dialogs/richtext-editor/plugin-components/index.md

```diff
@@ -52,458 +53,20 @@ Where:
-<p>
-
-<p>
-
-   </td>
-  </tr>
-  <tr>
-   <td>HardBreak
-   </td>
-   <td><strong>wcm/dialogs/components/richtext/plugin/hardbreak</strong>
-   </td>
-   <td>{}
-   </td>
-   <td>getAction:
-<p>
-<code>1{ </code>
-<p>
-<code>2  execute ({}) => {...}</code>
-<p>
-<code>3}</code>
-<p>
-getState:
-<p>
-<code>1{}</code>
-   </td>
-   <td><code>1&lt;plugin jcr:primaryType="nt:unstructured"</code>
-<p>
-<code>2        sling:resourceType="wcm/dialogs/components/richtext/plugin/hardbreak"/></code>
-   </td>
-  </tr>
-  <tr>
-   <td>TextAlign
-   </td>
-   <td><strong>wcm/dialogs/components/richtext/plugin/textalign</strong>
-   </td>
-   <td><code>{ alignment }</code>
-<p>
-level - <code>string (default: 'left')</code> - Text alignment ('left', ‘center', ‘right',  'justify')
-<p>
-
-   </td>
-   <td><br>getAction:
-<p>
-<code>1{ </code>
-<p>
-<code>2  execute ({}) => {...}</code>
-<p>
-<code>3}</code>
-<p>
-getState:
-<p>
-<code>1{ </code>
-<p>
-<code>2  isActive: ..., // define if current content has proper alignment.</code>
-<p>
-<code>3}</code>
-   </td>
-   <td><code>1&lt;plugin jcr:primaryType="nt:unstructured"</code>
-<p>
-<code>2        sling:resourceType="wcm/dialogs/components/richtext/plugin/textalignment"</code>
-<p>
-<code>3        alignment="right"/></code>
-<p>
-
-   </td>
-  </tr>
-  <tr>
-   <td>Undo
-   </td>
-   <td><strong>wcm/dialogs/components/richtext/plugin/undo</strong>
-   </td>
-   <td><code>{}</code>
-   </td>
-   <td><br>getAction:
-<p>
-<code>1{ </code>
-<p>
-<code>2  execute ({}) => {...}</code>
-<p>
-<code>3}</code>
-<p>
-getState:
-<p>
-<code>1{ </code>
-<p>
-<code>2  isDisabled: ..., // define if action can be run.</code>
-<p>
-<code>3}</code>
-   </td>
-   <td><code>1&lt;plugin jcr:primaryType="nt:unstructured"</code>
-<p>
-<code>2        sling:resourceType="wcm/dialogs/components/richtext/plugin/undo"/></code>
-<p>
-
-   </td>
-  </tr>
-  <tr>
-   <td>Redo
-   </td>
-   <td><strong>wcm/dialogs/components/richtext/plugin/redo</strong>
-   </td>
-   <td>{}
-   </td>
-   <td>getAction:
-<p>
-<code>1{ </code>
-<p>
-<code>2  execute ({}) => {...}</code>
-<p>
-<code>3}</code>
-<p>
-getState:
-<p>
-<code>1{ </code>
-<p>
-<code>2  isDisabled: ..., // define if action can be run.</code>
-<p>
-<code>3}</code>
-   </td>
-   <td><code>1&lt;plugin jcr:primaryType="nt:unstructured"</code>
-<p>
-<code>2        sling:resourceType="wcm/dialogs/components/richtext/plugin/redo"/></code>
-<p>
-
-   </td>
-  </tr>
-</table>
+| Name            | Path                                                            | Configuration                                                                                                                                            | Interfaces                                                                                                                            | Example Usage                                                                                                        |
```

Comment:
Maybe lets merge name and path column (name and path below) and name it Plugin?
Configuration is empty for most of plugins. Myabe instead of separated column it could be mentiobed in usage column?
Is the information in Interfaces column needed for users? Maybe should be removed?
Those changes would create more space for the usage column